x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[security-domain]]
+=== Security domains
+
+beta::[]
+
+Security domains are a method of grouping <<realms,realms>> to enable resource
+sharing across different realms for the same user.
+
+==== How security domain works
```

Comment:
```suggestion
==== How security domains work
```

Alternatively "How __*a*__ security domain works", but that doesn't read as well.